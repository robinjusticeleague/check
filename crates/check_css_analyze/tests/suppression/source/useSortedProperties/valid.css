/* should not generate diagnostics */
/* check-ignore assist/source/useSortedProperties: test */
.not-sorted-basic {
	color: red;
	display: block;
}
