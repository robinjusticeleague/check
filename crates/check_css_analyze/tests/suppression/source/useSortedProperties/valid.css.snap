---
source: crates/check_css_analyze/tests/spec_tests.rs
expression: valid.css
---
# Input
```css
/* should not generate diagnostics */
/* check-ignore assist/source/useSortedProperties: test */
.not-sorted-basic {
	color: red;
	display: block;
}

```
